python/ray/_private/function_manager.py

```diff
@@ -102,6 +103,7 @@ def __init__(self, worker):
         self._num_exported = 0
         # This is to protect self._num_exported when doing exporting
         self._export_lock = threading.Lock()
+        RemoteFunctionWhitelist.whitelist_init()
```

Comment:
We can automatically do this when importing the module.

Reply:
Done